Tracker/Modules/MainScreenModule/TrackerModule/TrackerMainScreen/TrackersViewController.swift

```diff
@@ -130,24 +111,33 @@ final class TrackersViewController: UIViewController & TrackerToCoordinatorProto
         setupConstraints()
         setupNavigationAttributes()
         
-        // загрузка трекеров и записей
-        fetchTrackers()
-        fetchRecords()
-        
-        
-        checkForScheduledTrackers()
-        checkForEmptyState()
+        bind()
+        checkTheCollectionViewState()
         
     }
     
     //MARK: Methods
     
-    private func fetchTrackers() {
-        visibleCategories = coreDataManager.fetchedCategories
+    private func bind() {
+        viewModel?.$emptyStackViewIsHidden.bind(action: { [weak self] isHidden in
+            guard let self else { return }
+            self.emptyStateStackView.isHidden = isHidden
```

Comment:
Если код короткий и нет необходимости манипулировать self-свойствами (вычислять, переприсваивать, передавать), стоит отдавать предпочтение короткому варианту 

```suggestion
            self?.emptyStateStackView.isHidden = isHidden
```